pyproject.toml

```diff
@@ -81,6 +81,7 @@ server = [
     "fastapi>=0.109",  # For Python 3.12 support
 ]
 typing = [
+    "numpy>=2.2.2",  # Pin for np.float typing fixes
```

Comment:
Can we loosen this and sacrifice typing? IIRC the >=1 pin came from a lack of universal numpy 2 support. I think internal code will break if this is merged. 

I don't remember who/what hit this issue, but we ought to figure it out and add in a comment here.

Reply:
Sure, we don't _need_ this pin, it's just a nice to have, so I reverted the commit